README.md

```diff
@@ -276,11 +276,15 @@ To run the tests, identify your image tag from the build step and run `make test
 CONTAINER=namely/protoc-all:VVV make test
 ```
 
-(`VVV` is your version from the tag in the console output when running `make build`). Running this will
+(`VVV` is your version from the tag in the console output when running `make build`.) Running this will
 demonstrate that your new image can successfully build containers for each language.
 
 ### Release
 
+```sh
+$ make push
+```
+
```

Comment:
this isn't manually used anymore. just through the CI pipeline

Reply:
Should I delete (or at least comment out) the `push*` targets, to avoid future confusion?

Reply:
you mean in the makefile? We should keep it because it's being used by the CI pipeline, for example here - https://github.com/namely/docker-protoc/blob/b73892ba03b7a421995c613db8e9818115d3b6d1/.github/workflows/release.yml#L49

Reply:
Okay, I added a note to that effect, in place of my previous addition.